### PR TITLE
Release notes ccm 21 march

### PR DESCRIPTION
**Pull request by charanya-j**
Added release notes for March 21 release

### CODE REVIEW

File: release-notes/cloud-cost-management.md [hunk 1]
```diff
@@ -13,6 +13,32 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+  With this enhancement, you can track all CRUD operations such as Create, Delete, and Update related to budget groups.
+
+* Display the AWS account ID and name on the **Recommendations** page. (CCM-11666)
+
+  The AWS Account ID has been added to ECS Service on the Recommendations list page.
+
+* Cost category enhancement (CCM-10580)
+
+  Introduced support to allocate the cost of shared cost buckets by a fixed percentage among each cost bucket. For more information, go to [Use Cost Categories](/docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md).
```
Comment (sdsingh07) [inline release-notes/cloud-cost-management.md:29]: We can add that we have launched new UI for cost category.

Comment (charanya-j) [reply release-notes/cloud-cost-management.md]: Thank you, Sahil. Added this info.

File: docs/cloud-cost-management/2-use-cloud-cost-management/ccm-currency-preferences/currency-preferences.md [hunk 2]
```diff
@@ -79,6 +79,7 @@ Once configured, you cannot change the currency settings. It takes up to 24 hour
 
   ![](./static/currency-preference-recommendations.png)
 
+* Currency standardization is supported only within CCM. Note that the currency symbol in **Dashboards** is not changed, but the cost is displayed in the preferred currency.
```
Comment (SmritiSatya) [inline docs/cloud-cost-management/2-use-cloud-cost-management/ccm-currency-preferences/currency-preferences.md:82]: Note that the currency symbol in **Dashboards** has not changed, but the cost is displayed in the preferred currency.

File: release-notes/cloud-cost-management.md [hunk 3]
```diff
@@ -13,6 +13,37 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 21, 2023, version 78903
+
+### What's new
+* Enabled audit trail for budget groups. (CCM-11387)
+
+  With this enhancement, you can track all CRUD operations such as Create, Delete, and Update related to budget groups.
```
Comment (SmritiSatya) [inline release-notes/cloud-cost-management.md:21]: With this enhancement, you can track all CRUD (Create, Delete, Update, and Delete) operations related to budget groups.

File: release-notes/cloud-cost-management.md [hunk 4]
```diff
@@ -13,6 +13,37 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  
+  For more information, go to [Use Cost Categories](https://developer.harness.io/docs/cloud-cost-management/use-cloud-cost-management/ccm-cost-categories/use-ccm-cost-categories).
+
+### Early access
+This release does not include any early access features.
+
+### Fixed issues
+* The error message displayed while creating a Jira ticket to apply recommendations was not meaningful. (CCM-10822)
+
+  A comprehensive Jira error message will be displayed in the user interface whenever it is feasible. However, there may be instances where only a generic message, such as "Error creating issue," will be displayed when the Jira error is not parsed.
```
Comment (SmritiSatya) [inline release-notes/cloud-cost-management.md]: A comprehensive Jira error message will be displayed in the user interface whenever feasible. However, there may be instances where only a generic message, such as "Error creating issue," will be displayed when the Jira error is not parsed.

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**